sdk/resourcemanager/azure-resourcemanager-redis/src/main/java/com/azure/resourcemanager/redis/models/RedisCache.java

```diff
@@ -581,6 +585,23 @@ enum RedisVersion {
         public String getValue() {
             return value;
         }
+
+        /**
+         * Gets the major version from the given redis version.
+         * @param redisVersion valid redis version, e.g. 6.0.14
+         * @return the major redis version, or null if the given redisVersion is invalid
+         */
+        public static RedisVersion fromVersion(String redisVersion) {
```

Comment:
If it is only used in test, should we add a public method here?

Reply:
Our `RedisCache.redisVersion()` returns minor version string, e.g. 6.x.x. Maybe we can add a convenience method for converting minor version to major version?